packages/typeorm/README.md

```diff
@@ -0,0 +1,145 @@
+`TypeOrmComponent` component. Import them from `@loopback/typeorm` and configure
+your app as shown below.
+
+```ts
+import {TypeOrmComponent, TypeOrmMixin} from '@loopback/typeorm';
+...
+export class AppaApplication extends BootMixin(
+  ServiceMixin(TypeOrmMixin(RestApplication)),
+) {
+  constructor(options: ApplicationConfig = {}) {
+    super(options);
+    ...
+    this.component(TypeOrmComponent);
+    ...
+  }
+}
+
+```
+
+### Creating Entities
+
+[Entities](https://typeorm.io/#/entities) are equivalent to Juggler's Models.
+Define the entities as usual and keep them in a directory named `entities`, or
+even `models` or any name, just make sure to load them correctly in the
+connections.
+
+```ts
+// src/entities/book.entity.ts
+import {Entity, Column, PrimaryColumn} from 'typeorm';
+@Entity()
+export class Photo {
+  @PrimaryColumn()
+  id: number;
+
+  @Column()
+  title: string;
+
+  @Column()
+  isPublished: boolean;
+}
+```
+
+### Creating Connections
+
+[Connections](https://typeorm.io/#/connection) are equivalent to Juggler
+datasources. They contain the connectivity and other details about the
+databases. Define the connection in files with a `.connection.ts` extension and
+keep them in the `connections` directory of the project.
+
+```ts
+// src/connections/sqlite.connection.ts
+import path from 'path';
+import {SqliteConnectionOptions} from '@loopback/typeorm';
+import {Book} from '../entities/';
+
+export const SQLLITE_CONNECTION: SqliteConnectionOptions = {
+  type: 'sqlite',
+  database: path.join(__dirname, 'sqlite-test-db'),
+  entities: [Book],
+  synchronize: true,
+};
+```
+
+Make sure to install the underlying database driver. For example, if you are
+using SQlite, you'll need to install `sqlite3`.
+
+```sh
+npm install sqlite3
+```
+
+Refer to the
+[TypeORM documentation](https://github.com/typeorm/typeorm#installation) for the
+supported databases and the underlying drivers.
+
+### Creating controllers
+
+Controllers continue to work as usual. And you don't have to create repositories
+since TypeORM creates them for you; just inject them in the controllers. The
+repository API is 100% TypeORM
+[repository API](https://typeorm.io/#/repository-api).
+
+```ts
+// src/controllers/book.controller.ts
+import {get, post, Request, requestBody} from '@loopback/rest';
+import {getModelSchemaRef, Repository, typeorm} from '@loopback/typeorm';
+import {Book} from '../entities';
+
+export class BookController {
+  @typeorm.repository(Book) private bookRepo: Repository<Book>;
+
+  constructor() {}
+
+  // Create a new book
+  @post('/book')
+  async create(@requestBody() data: Book) {
+    const book = new Book();
+    book.title = data.title;
+    book.published = false;
+    return await this.bookRepo.save(book);
+  }
+
+  // Find book by title
+  @get('/note/{title}')
+  async findByTitle(@param.path.string('title') title: string) {
+    return await this.bookRepo.find({title});
```

Comment:
Please add a comment to make it clear to readers that we are using TypeORM filtering syntax here, not LoopBack's style.

Based on what I found in the docs ([Find Options](https://typeorm.io/#/find-options)), I think this line should use `where`.

```suggestion
    return await this.bookRepo.find({where:{title}});
```

Reply:
TypeORM syntax looks like `"title": title`, `name: "Yahoo"`.

Reply:
Weird, their documentation is showing something else. https://typeorm.io/#/find-options

Did you verify that your implementation returns only the books with the given title and no other book entries?

I don't mind how we write the filter as long as it works.